src/sparsezoo/model/model.py

```diff
@@ -259,6 +257,8 @@ def path(self) -> str:
 
         # extract onnx_model if it is a tar.gz
         self.onnx_model.path
+        # extract deployment directory if it is a tar.gz
+        self.deployment_directory_path
```

Comment:
We need this line of code if we wish to untar `deployment.tar.gz` after we execute `Model(stub).download()` (see the example in the PR description)